project_event/report/project_task_activity_report.py

```diff
@@ -0,0 +1,105 @@
+# © 2019 Savoir-faire Linux
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+from odoo import models, api, _
+from datetime import datetime, timedelta
+from odoo.tools import DEFAULT_SERVER_DATE_FORMAT as DATE_FORMAT
+from operator import itemgetter
+
+
+class ReportWeekly(models.AbstractModel):
+    _name = 'report.project_event.project_task_activity_report_view'
+
+    title = _("Activity's work slips")
+    print_date_text = _("Print Date")
+    activity_label = _("Activity: ")
+    date_label = _("Date: ")
+    start_hour_label = _("Hour: ")
+    end_hour_label = _("Expected End Time: ")
+    number_spectators_label = _("Number of spectators: ")
+    client_label = _("Client: ")
+    contact_label = _("Contact 1 :")
```

Comment:
why contact 1?

Reply:
Because there will also be Contact 2, specs